.github/workflows/python-app.yml

```
@@ -32,3 +32,8 @@ jobs:
 
       - name: Run linting and tests
         run: make ci
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)